files/initramfs-tools/arista-convertfs.j2

```diff
@@ -82,6 +82,24 @@ run_cmd() {
     fi
 }
 
+create_varlog_file() {
```

Comment:
>create_varlog_file [](start = 0, length = 18)

This function is not Arista specific. Suggest to move to a new file or into union-mount.